src/ray/common/common_protocol.h

```diff
@@ -104,15 +104,13 @@ string_vec_to_flatbuf(flatbuffers::FlatBufferBuilder &fbb,
 template <typename ID>
 flatbuffers::Offset<flatbuffers::String> to_flatbuf(flatbuffers::FlatBufferBuilder &fbb,
                                                     ID id) {
-  return fbb.CreateString(reinterpret_cast<const char *>(id.data()), sizeof(ID));
+  return fbb.CreateString(reinterpret_cast<const char *>(id.data()), kUniqueIDSize);
```

Comment:
Though it is the same, `kUniqueIDSize`->`id.size()` looks better? However, we cannot use `.size()` in `from_flatbuf`, which makes it a bit strange...